### PR TITLE
Add typescript types for returned tokens

### PR DESCRIPTION
**Pull request by jaclarke**

### CODE REVIEW

File: src/interfaces.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+export interface Position {
+  line: number;
+  column: number;
+  offset: number;
+}
+
+export enum TokenKind {
+  Assign = "Assign",
```
Comment (1st1) [inline src/interfaces.ts:14]: @tailhook I'm curious if there's a way to reflect these automatically...

Comment (tailhook) [reply src/interfaces.ts:14]: Yes, kinda: https://github.com/edgedb/edgedb-js-lexer/pull/4
This generates:
```typescript
export enum TokenKind {
  Assign,
  SubAssign,
  AddAssign,
  // ...
```
So without string values. And we still can't export list of our own classes from wasm. So it must be casted with `TokenKind[token.kind]`. But it's better than supporting the separate list anyway.

Comment (1st1) [reply src/interfaces.ts:14]: @jaclarke James, I've merged Paul's #4 PR. Please take a look if you need anything else. IMO, a non-string enum should be OK.

**Comment by 1st1** [comment]
See https://github.com/edgedb/edgedb-js-lexer/pull/2/files#r484564585

File: src/interfaces.ts (hunk 1 above)
Comment (jaclarke) [reply src/interfaces.ts:14]: @tailhook Do you know if there's a way to export `TokenKind` into it's own file, separate from the main `edgeql_wasm_bg.js` file? There may not be, I can only find an option to set the name of the export in the wasm-bindgen docs, in which case I can just extract it in some post build step.

Comment (tailhook) [reply src/interfaces.ts:14]: I can't find any way to do that.

**Pull request closed without merging**